.github/workflows/zap_regeneration.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
 
         runs-on: ubuntu-20.04
         container:
-            image: connectedhomeip/chip-build:0.6.28
+            image: connectedhomeip/chip-build:0.6.30
```

Comment:
since this changes the docker image, in #24307 we discovered that zap may need updates for cert bins as well.

Update the zap image in `integrations/docker/images/chip-cert-bins/Dockerfile` as well?

Reply:
Done.